_posts/2021-04-09-prace-workshop.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: MDAnalysis Workshop @ PRACE / SURF
+---
+
+MDAnalysis will be delivering a 3 half-day remote workshop hosted by PRACE / SURF on the 26-28th of May.
+
+The workshop will include both lecture and practical portions. We will cover topics including:
+
+* The fundamentals and basics of MDAnalysis
+* How to use existing analyses
+* How to write your own analyses
+* Parallel analysis in an HPC setting
+* Advanced functionality such as universe construction and transformations
+
+
+## Register soon [places limited]
+
+The workshop is open to everyone, but places are limited, apply early!
```

Comment:
```suggestion
Places are limited. Please register early! You can sign up for individual sessions if you are only interested in certain topics.
```

Reply:
Is that true? I thought you had to sign up to the whole thing.

Reply:
Oh. Err I thought it was. Let's leave it as a suggestion for now

Reply:
I can't see anything on the registration form that allows you to do that, thoughts @mnmelo ?

Reply:
I don't think we can do separate registrations.